src/main/java/com/google/api/codegen/DiscoveryFragmentGeneratorApi.java

```diff
@@ -69,6 +70,13 @@
           "The list of YAML configuration files for the fragment generator.",
           ImmutableList.<String>of());
 
+  public static final Option<String> AUTH_INSTRUCTIONS_URL =
+      ToolOptions.createOption(
+          String.class,
+          "auth_instructions",
+          "An @-delimited map of language to auth instructions URL: lang:URL@lang:URL@...",
```

Comment:
I think `:` makes more sense to indicate mapping... I'm going to leave it as is and this whole thing will disappear on the other branch.